frontend/src/App.vue

```diff
@@ -1,32 +1,48 @@
+
 <template>
-  <v-app>
-    <v-toolbar app>
-      <v-toolbar-title class="headline text-uppercase">
-        <span>Vuetify</span>
-        <span class="font-weight-light">MATERIAL DESIGN</span>
-      </v-toolbar-title>
-      <v-spacer></v-spacer>
-      <v-btn
-        flat
-        href="https://github.com/vuetifyjs/vuetify/releases/latest"
-        target="_blank"
-      >
-        <span class="mr-2">Latest Release</span>
-        <v-icon>open_in_new</v-icon>
-      </v-btn>
+    <v-app>
+    <v-toolbar>
+        <v-toolbar-title class="headline text-uppercase">
+        <span>Amazon Dash</span>
+        </v-toolbar-title>
     </v-toolbar>
-
-    <v-content>
-      <router-view/>
+    <v-content app>
+    <div class="container">
+        <div class="row">
```

Comment:
Move this code into a separate template file, something like `src/views/Services.vue`, and replace the `<router-view/>` tag. Then add your template to the router with a path such as `/services`. Go look at `src/router.ts`, and `src/views/Home.vue` for example.